core/templates/dev/head/pages/Base.js

```diff
@@ -28,6 +28,7 @@ oppia.controller('Base', [
     $scope.siteFeedbackFormUrl = GLOBALS.SITE_FEEDBACK_FORM_URL;
     $scope.promoBarIsEnabled = GLOBALS.PROMO_BAR_IS_ENABLED;
     $scope.promoBarMessage = GLOBALS.PROMO_BAR_MESSAGE;
+    $scope.version = GLOBALS.explorationVersion;
```

Comment:
Base.js is used in non-exploration pages. You'll want to define the version only in the exploration player page, so maybe using its controller would be better.